### PR TITLE
fix(components): accordion component throws "validateDOMNesting" erro…

### LINKED ISSUE #2168: [BUG] - Accordion component throws "validateDOMNesting" error on the storybook.
### NextUI Version

2.0.0

### Describe the bug

When you open up the Accordion component(Custom with classNames) on the storyboard, you will get an error console message that says "validateDOMNesting(...) cannot appear as a descendant of p".(the pic below)

error

### Your Example Website or App

https://storybook.nextui.org/?path=/story/components-accordion--custom-with-class-names

### Steps to Reproduce the Bug or Issue

1. Go to the storyboard.
2. Go to the Accordion component(Custom with classNames).

### Expected behavior

No console error would be ideal.

### Screenshots or Videos

_No response_

### Operating System Version

macOS

### Browser

Chrome

### PR DESCRIPTION
**Pull request by kuri-sun**
Closes #2168

## 📝 Description

The accordion component throws a "validateDOMNesting" error on the storybook

## ⛳️ Current behavior (updates)

When you open up the Accordion component(Custom with classNames) on the storyboard, you will get an error console message that says "validateDOMNesting(...) cannot appear as a descendant of p".(the pic below)
error

## 🚀 New behavior

There are no errors in the console.

## 💣 Is this a breaking change (Yes/No):

No.

### CODE REVIEW

File: .changeset/silver-walls-drum.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (jrgarciadev) [inline .changeset/silver-walls-drum.md:1]: @kuri-sun please remove this file, it is not needed for storybook's changes.

**Review by jrgarciadev — CHANGES_REQUESTED** [review]

**Comment by jrgarciadev** [comment]
@kuri-sun sorry, could you please modify this component code as well? https://nextui.org/docs/components/accordion#start-content

**Comment by kuri-sun** [comment]
@jrgarciadev Absolutely

**Comment by kuri-sun** [comment]
@jrgarciadev The requested change was modified. Thank you! 😄

**Comment by jrgarciadev** [comment]
@kuri-sun could you please update this as well?  https://vscode.dev/github/nextui-org/nextui/blob/main/apps/docs/content/components/accordion/custom-styles.ts#L217

**Comment by kuri-sun** [comment]
@junior Of course, immediately!

**Review by jrgarciadev — APPROVED** [review]

**Pull request merged**